content/src/main/content/jcr_root/apps/apm/clientlibs/dashboard/js/apm-dashboard.js

```diff
@@ -93,13 +97,26 @@
                                     checkStatus(jobId)
                                 }, 1000);
                             } else if (dataObject.type === 'finished') {
-                                console.log(scriptPath + " finished: " + JSON.stringify(dataObject.entries));
+                                let status = getResponseStatus(dataObject),
+                                    title;
+
                                 switch(mode){
                                     case 'DRY_RUN':
-                                        uiHelper.notify('info', 'Dry Run executed successfully', 'Info');
+                                        title = 'Dry Run';
                                         break;
                                     case 'RUN':
-                                        uiHelper.notify('info', 'Run on author executed successfully', 'Info');
+                                        title = 'Run on author';
+                                        break;
+                                }
+                                switch (status) {

```

Comment:
minor: inconsistent formatting (probably it should be changed at line 104).

Reply:
I'm not getting what is wrong. Can You explain?

Reply:
spaces around the switch statement :)